src/core/base/VaultBase.sol

```diff
@@ -335,6 +352,12 @@ abstract contract VaultBase is Controllable, ERC20Upgradeable, ReentrancyGuardUp
 
     }
 
+    /// @inheritdoc IVault
+    function getUniqueInitParamLength() external view returns(uint uniqueInitAddresses, uint uniqueInitNums) {
+        VaultBaseStorage storage $ = _getVaultBaseStorage();
+        return (IVault($.strategy.vault()).UNIQUE_INIT_ADDRESSES(), IVault($.strategy.vault()).UNIQUE_INIT_NUMS());
```

Comment:
implement base NIQUE_INIT_ADDRESSES() public function without implementation in this base contract
and write `UNIQUE_INIT_ADDRESSES()` instead that your code